packages/localize-tools/src/types/modes.d.ts

```diff
@@ -11,21 +11,32 @@ export interface RuntimeOutputConfig {
   mode: 'runtime';
 
   /**
-   * Output directory for generated TypeScript modules. Into this directory will
-   * be generated a <locale>.ts for each `targetLocale`, each a TypeScript
-   * module that exports the translations in that locale keyed by message ID.
+   * Language for emitting generated modules. Options:
+   *
+   * - "ts": Default. Emit TypeScript modules with ".ts" file extension.
```

Comment:
Yeah, as with the other comment, I think js should be the default unless there's a tsconfig, although... maybe that default can change in a tool that wraps the transformer for usage outside TS toolchains?

Reply:
Done.